### DISCUSSION TITLE
How to apply a force on a body in Ferrite.jl ?

### DISCUSSION THREAD

**Discussion by WaremF** [Ferrite-FEM/Ferrite.jl#709] [General]
Hello,
I'm new in Ferrite.jl und would like to know how I can apply a tensile force in on a square. For example, let's say I have a square (a=1 mm). And it should be fixed on the left side (u=0). On the right side I want to apply a distributed tensile force (1 N/mm) see figure. How do I define this force In Ferrite?
thanks
Warem

![tensileTest](https://github.com/Ferrite-FEM/Ferrite.jl/assets/77281990/4ed8f1dc-9b53-4516-bd64-571d8d1b0a3b)

**Comment by fredrikekre** [comment]
Hi, and welcome!
That would normally correspond to a Neumann boundary condition. This section of the manual https://ferrite-fem.github.io/Ferrite.jl/stable/manual/boundary_conditions/#Neumann-Boundary-Conditions amd this example https://ferrite-fem.github.io/Ferrite.jl/stable/examples/incompressible_elasticity/ will hopefully get you started. Don't hesitate to ask if something needs to be clarified 🙂

**Comment by WaremF** [comment]
thank you for the quick reply
